### DISCUSSION TITLE
How to fold code blocks

### DISCUSSION THREAD

**Discussion by ffabiorj** [LunarVim/LunarVim#3909] [General]
How can I fold a code block in Lvim? E.g JSON files.
In Vscode I use Ctrl+Shift+[

**Comment by igson** [comment]
backspace  + gb

**Reply by ffabiorj** [reply]
it doesn't work for me.

**Reply by igson** [reply]
![Captura de tela de 2023-03-01 16-19-46](https://user-images.githubusercontent.com/4922513/222256470-d8ed06a9-9227-49b3-b83b-8f6b3c75756c.png)

**Reply by ffabiorj** [reply]
I think you got me wrong.

I'd to do this like the picture.
![picture1](https://user-images.githubusercontent.com/6344265/222266552-b42c18cb-366d-4d95-933b-2eb358cd93d4.png)
![picture2](https://user-images.githubusercontent.com/6344265/222266589-1bd399f8-80b5-43a6-a650-05db046bdb84.png)

**Comment by igson** [comment]
Select the content and press gb

**Comment by oscardoso** [comment]
try this Q&A #3940
